### PR TITLE
Add mahotas & imread

### PR DESCRIPTION
**Pull request by luispedro**
This is a follow up to PR #11670 which I accidentally deleted.

/cc @domenkozar @FRidh

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3097,6 +3097,34 @@ in modules // {
...
+      numpy
+      imread
+    ];
+
+    meta = with stdenv.lib; {
+      description = "Computer vision package based on numpy";
+      homepage = https://readthedocs.org/projects/mahotas/;
+      maintainers = with maintainers; [ luispedro ];
+      license = licenses.mit;
+      platforms = platforms.linux;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: is it only supposed to work on linux or also darwin/osx?

Comment (luispedro) [inline pkgs/top-level/python-packages.nix]: I cannot test on those platforms at all, so I didn't want to claim the build script works, but I do know that mahotas itself is commonly run there.

Comment (luispedro) [inline pkgs/top-level/python-packages.nix]: Should I add them? What is the protocol here? Thanks.

Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: It's a good reason. Not sure what the protocol is actually . If I'm correct this field is generally only included when it's known *not* to work on a certain platform; the default value includes both platforms. I don't think it's too much of a problem; if a darwin user sees this, they can always change it.

**Comment by FRidh** [comment]
@luispedro 
It seems there are some tests that fail when building mahotas on 32-bit system:
http://hydra.nixos.org/eval/1233470?filter=python&compare=1233391&full=#tabs-still-fail

**Comment by luispedro** [comment]
Thanks. This is an upstream problem. Fortunately, I'm also the upstream maintainer, so I will fix it there first and then update the nixpgks version.

**Pull request merged**